packages/manager/modules/pci/src/projects/project/activate/translations/Messages_fr_FR.json

```
@@ -0,0 +1,15 @@
+{
+  "pci_projects_project_breadcrumb": "Activation projet",
+  "pci_projects_project_activate_title": "Activer votre projet",
+  "pci_projects_project_activate_description": "La validation d'un moyen de paiment est obligatoire pour activer votre projet. Si vous choisissez de ne consommer aucune ressource Public Cloud, vous ne serez pas débité.",
+  "pci_projects_project_activate_cta": "Activer votre projet",
+
+  "pci_projects_project_pci_banner_promotion_title": "200€ offerts",
+  "pci_projects_project_pci_banner_promotion_description_1": "Valable 1 mois pour tester nos services après l'ouverture de votre projet.",
+  "pci_projects_project_pci_banner_promotion_description_2": "Vous serez débité uniquement si vous poursuivez l'utilisation de nos produits au delà de cette somme ou cette date. De quoi démarrer sereinement.",
+  "pci_projects_project_pci_banner_section_1_title": "Facturation granulaire",
+  "pci_projects_project_pci_banner_section_1_description": "La facturation des ressources se fait en fonction de votre utilisation. Si vous utiliser un produit une heure, vous ne serez facturée que de cette heure là.",
+  "pci_projects_project_pci_banner_section_2_title": "Facturation simple",
+  "pci_projects_project_pci_banner_section_2_description": "Pas de coûts cachés et de surprise en fin de mois. Notre grille tarifaire est construite pour être compréhensible.",
```

```suggestion
  "pci_projects_project_pci_banner_section_2_description": "Pas de coût caché et de surprise en fin de mois. Notre grille tarifaire est construite pour être compréhensible.",
```
Ca ne devrait pas être au singulier ?